pulsar-functions/utils/src/test/java/org/apache/pulsar/functions/utils/FunctionConfigUtilsTest.java

```diff
@@ -157,6 +157,15 @@ public void testMergeDifferentInputs() {
         FunctionConfigUtils.validateUpdate(functionConfig, newFunctionConfig);
     }
 
+    @Test(expectedExceptions = IllegalArgumentException.class, expectedExceptionsMessageRegExp = "Input Specs mismatch")
```

Comment:
Can we also add a test case that tests that other configs e.g. receiverQueueSize can be passed/updated without error

Reply:
added